index.bs

```diff
@@ -10877,35 +10877,54 @@ defined on that interface.
 These properties are described in more detail
 in sections [[#es-attributes]] and [[#es-operations]].
```

Comment:
This paragraph was untouched, but the paragraphs about constants and the constructor property were deleted. Maybe best to delete them all, or alternately convert them into a brief non-normative summary (e.g. as a buletted list in a NOTE)?